pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -408,6 +427,18 @@ func (a Adapter) execDevfile(commandsMap common.PushCommandsMap, componentExists
 	return
 }
 
+func (a Adapter) execTestCmd(testcmd versionsCommon.DevfileCommand, podName string, containers []corev1.Container, show bool) (err error) {
+	compInfo := common.ComponentInfo{
+		PodName: podName,
+	}
+	compInfo.ContainerName = testcmd.Exec.Component
+	err = exec.ExecuteDevfileCommands(&a.Client, *testcmd.Exec, testcmd.Exec.Id, compInfo, show, a.machineEventLogger)
+	if err != nil {
+		return err
+	}
+	return
```

Comment:
you can just do `return err` here, no need for the if block since its returning right after

Reply:
Updated